apps/site/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from "./Button.module.css";
+
+interface ButtonProps {
+	text: string;
+	href?: string;
+	isLink: boolean;
+}
+
+const Button: React.FC<ButtonProps> = ({ text, href, isLink }) => {
+	if (isLink) {
+		return (
+			<a
+				href={href}
+				target="_blank"
+				rel="noopener noreferrer"
+				className={styles.button + " font-body"}
+			>
+				{text}
+			</a>
+		);
+	}
+	return (
+		<input
+			type="submit"
```

Comment:
Note: think using an actual `button` element would be more appropriate, we can revisit this.